sdks/python/apache_beam/io/gcp/bigquery_file_loads.py

```diff
@@ -515,9 +515,17 @@ def start_bundle(self):
       self.bq_io_metadata = create_bigquery_io_metadata(self._step_name)
     self.pending_jobs = []
 
-  def process(self, element, job_name_prefix=None, unused_schema_mod_jobs=None):
-    destination = element[0]
-    job_reference = element[1]
+  def process(
+      self, element_list, job_name_prefix=None, unused_schema_mod_jobs=None):
+    if isinstance(element_list, tuple):
+      # Allow this for streaming update compatibility while fixing BEAM-24535.
+      self.process_one(element_list)
+    else:
+      for element in element_list:
+        self.process_one(element)
```

Comment:
Let's pass the side inputs into `process_one`, this is probably causing the failing tests in https://ci-beam.apache.org/job/beam_PreCommit_Python_Phrase/3289/testReport/

Reply:
Yeah, I fixed this, but not it's failing on (unrelated-looking) proto issues. Trying to run tests again...